pkg/api/server/register_images.go

```diff
@@ -659,10 +655,71 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	// - application/json
 	// responses:
 	//   200:
-	//     description: no error
+	//     $ref: "#/response/LibpodImagesLoadResponse"
+	//   500:
+	//     $ref: '#/responses/InternalError'
+	r.Handle(VersionedPath("/libpod/images/load"), APIHandler(s.Context, libpod.ImagesLoad)).Methods(http.MethodPost)
+	// swagger:operation POST /libpod/images/import libpod libpodImagesImport
+	// ---
+	// tags:
+	//  - images
+	// summary: Import image
+	// description: Import a previosly exported tarball as an image.
+	// parameters:
+	//  - in: query
+	//    name: change
+	//    description: "Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR.  JSON encoded string"
+	//    type: string
+	//  - in: query
+	//    name: message
+	//    description: Set commit message for imported image
+	//    type: string
+	//  - in: query
+	//    name: url
+	//    description: Specify a URL instead of a tarball
+	//    type: bool
+	//  - in: body
+	//    name: request
+	//    description: Tarball of (or URL to) container image
+	//    required: true
+	//    schema:
+	//      type: string
+	// produces:
+	// - application/json
+	// responses:
+	//   200:
+	//     $ref: "#/response/LibpodImagesImportResponse"
+	//   500:
+	//     $ref: '#/responses/InternalError'
+	r.Handle(VersionedPath("/libpod/images/import"), APIHandler(s.Context, libpod.ImagesImport)).Methods(http.MethodPost)
+	// swagger:operation GET /libpod/images/pull libpod libpodImagesPull
+	// ---
+	// tags:
+	//  - images
+	// summary: Import image
+	// description: Import a previosly exported tarball as an image.
+	// parameters:
+	//  - in: query
+	//    name: reference
+	//    description: Mandatory reference to the image (e.g., quay.io/image/name:tag)/
+	//    type: string
+	//  - in: query
```

Comment:
Wait, aren't these a cookie?

Reply:
It depends what we want. We can also send a base64 encoded `DockerAuthConfig` (see c/images/types.DockerAuthConfig`).

However, since we're going through SSH to a remote machine, I don't think we need to pass around token.

Reply:
remember, you need to support both ssh and not.  This is not a remote api; it's an api (think cockpit)

Reply:
Let's make this a PS in today's meeting :+1: